app/views/pieces/show.html.haml

```diff
@@ -9,11 +9,28 @@
 %div= render 'reviews/form'
 
 - if !@reviews.nil?
-  %ul
+  %ul.clearfix#review_block
     - @reviews.each do |review|
       -if !review.title.empty?
-        %li= review.title
-        %li= review.body
-        %li= "written by: #{review.user.username}"
+        %div.review-container{id: "review_id-#{review.id}"}
+          %div.voting-title-container.clearfix
+            %div.voting{review_id: "#{review.id}"}
```

Comment:
`%div.voting` can be written as `.voting` since `%div` is implicit<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace